src/domain/resource/card/ResourceCard.js

```diff
@@ -68,7 +68,7 @@ class ResourceCard extends React.Component {
       intl,
       t,
     } = this.props;
-    const locale = intl.locale;
+    const locale = intl.locale === 'se' ? 'sv' : intl.locale;
```

Comment:
This is a hack, this issue should probably be fixed in `dataUtils.getLocalizedFieldValue()`.